modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -97,6 +105,11 @@ public class UpgradeCatchAllCheck {
 		CookiesManagerUtil.getCookieValue(name, httpServletRequest);
 	}
 
+	public void method(HttpServletRequest httpServletRequest, String name) {
+		CookieKeys.getCookie(httpServletRequest, "test123");
+		CookiesManagerUtil.getCookieValue(name, httpServletRequest);
+	}
```

Comment:
@MicaelleSilvaa, since the method above has the same parameters, why don't we reuse it?

Reply:
Resolved :)